integreat_cms/cms/utils/external_calendar_utils.py

```diff
@@ -245,6 +406,10 @@ def import_event(
         if previously_imported_event_translation
         else None
     )
+    previously_imported_recurrence_rule = RecurrenceRule.objects.filter(
```

Comment:
Why do we call it "previously_"? :) What is the context of this? :)

Reply:
I named it just like `previously_imported_event_translation` and `previously_imported_event` defined in the lines above, to indicated that this is the recurrence rule that was previously imported in this function (For example in the last run of the import_events management command). 
This is needed for the forms, e.g. so that they can check if it has changed through this import.

Reply:
Ah, okay, now I understand :) Thanks :)